### PR TITLE
Add Support for Lg Stylo 2 Plus [MetroPCS LGMS550]

### PR DESCRIPTION
**Pull request by messi2050**

### CODE REVIEW

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please

File: data/devices/lg.yml [hunk 1]
```diff
@@ -585,3 +585,53 @@
...
     pixel_format: RGBX_8888
     theme: portrait_hdpi
+    
+    
+- name: LG Stylo 2 Plus (MetroPCS)
+  id: ph2n
+  codenames:
+    - LGMS550
+    - ph2n
+  architecture:
```
Comment (chenxiaolong) [inline data/devices/lg.yml]: The `architecture` field cannot be empty.

Comment (messi2050) [inline data/devices/lg.yml]: Yea sorry for this it's armeabi-v7a

**Comment by messi2050** [comment]
Done architecture added

**Comment by messi2050** [comment]
Any more changes requested before merge ?

**Comment by chenxiaolong** [comment]
Sorry, I didn't see your changes. Looking now

**Comment by chenxiaolong** [comment]
@chenxiaolong-bot test this please

**Pull request merged**